.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,20 @@
+name: "CodeQL"
+
+on: [pull_request]
+jobs:
+  lint:
+    name: CodeQL
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+
+    - run: git checkout HEAD^2
+
+    - name: Run CodeQL
+      run: |
+        docker run --rm -v $PWD:/app composer sh -c \
+        "composer install --profile --ignore-platform-reqs && composer check
```

```suggestion
        "composer install --profile --ignore-platform-reqs && composer check"
```

You're missing the double quotes which is preventing the check from running